Package.swift

```diff
@@ -5,6 +5,7 @@ import PackageDescription
 let swiftSettings: [SwiftSetting] = [
     .enableExperimentalFeature("Embedded"),
     .enableExperimentalFeature("Volatile"),
+    .enableExperimentalFeature("ValueGenerics"),
```

Comment:
This is no longer needed in the latest toolchain.

https://github.com/swiftlang/swift/releases/tag/swift-DEVELOPMENT-SNAPSHOT-2025-03-14-a

Reply:
Removed. https://github.com/kkebo/swift_os/compare/9c2fd53e865afb3f52102a0d082c23654ef67ae2..948d5491b835695d8079b43d5671cb8f3b3b92f9